charts/ceph-csi-rbd/templates/nodeplugin-clusterrole.yaml

```diff
@@ -22,4 +22,7 @@ rules:
   - apiGroups: [""]
     resources: ["configmaps"]
     verbs: ["get"]
+  - apiGroups: [""]
```

Comment:
This should go to nodeplugin role instead of clusterrole?

Reply:
it needs to be a ClusterRole, the ServiceAccount is located in the Tenants namespace.